examples/unstructured_profilers.ipynb

```diff
@@ -0,0 +1,387 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f37ca393",
+   "metadata": {},
+   "source": [
+    "# Unstructured Profilers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff9bd095",
+   "metadata": {},
+   "source": [
+    "**Data profiling** - *is the process of examining a dataset and collecting statistical or informational summaries about said dataset.*\n",
+    "\n",
+    "The Profiler class inside the DataProfiler is designed to generate *data profiles* via the Profiler class, which ingests either a Data class or a Pandas DataFrame. \n",
+    "\n",
+    "Currently, the Data class supports loading the following file formats:\n",
+    "\n",
+    "* Any delimited  (CSV, TSV, etc.)\n",
+    "* JSON object\n",
+    "* Avro\n",
+    "* Parquet\n",
+    "* Text files\n",
+    "\n",
```

Comment:
Kind of a weird grey area, but since the Data class can take in dataframes and save them as data objects of a certain file type, does dataframe belong on this list?

Reply:
Just before I mention the profiler can take dataframes, and I'm not sure it would make sense to give them a middle step

Reply:
Nvm, I added it